fastly/resource_fastly_alert.go

```diff
@@ -57,6 +57,11 @@ func resourceFastlyAlert() *schema.Resource {
 				MaxItems:    1,
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
+						"ignore_below": {
+							Type:        schema.TypeFloat,
+							Optional:    true,
+							Description: "Floor noise that can be configured to ignore data points that are below this threshold.",
```

Comment:
Could we rewrite this description to be more like the API docs? I personally find the API description makes more sense:
https://www.fastly.com/documentation/reference/api/observability/alerts/definition/#evaluation-strategy-data-model

> Threshold for the denominator value used in evaluations that calculate a rate or ratio. Usually used to filter out noise.